PLEP-0007.rst

```diff
@@ -0,0 +1,266 @@
+| **tests**                                                                    |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/tests/``                                            |
+| :access: ``plasmapy.tests``                                                  |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of tests for top-level modules (i.e. functions and classes    |
+|   defined in top-level ``.py`` files).                                       |
++------------------------------------------------------------------------------+
+| .. _utils:                                                                   |
+|                                                                              |
+| **utils**                                                                    |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/utils/``                                            |
+| :access: ``plasmapy.utils``                                                  |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of "utility" functions and classes to help us write           |
+|   (what we try to think of as) clean, readable, and informative code.        |
+|                                                                              |
+|   This collection does not provide any physics tools, instead it is          |
+|   focused on providing package development tools.                            |
++------------------------------------------------------------------------------+
+
+.. +------------------------------------------------------------------------------+
+   | **examples**                                                                 |
+   +------------------------------------------------------------------------------+
+   | :directory: ``./plasmapy/examples/``                                         |
+   | :access: ``plasmapy.examples``                                               |
+   | :PLEP:                                                                       |
+   | :scope:                                                                      |
+   |   A collection of routines showing how to use and/or interface with the      |
+   |   various PlasmaPy tools.                                                    |
+   +------------------------------------------------------------------------------+
+
+Implementation
+==============
+
+*Describe the steps necessary to implement this PLEP, if necessary.*
+
+Issues, Pull Requests, and Branches
+===================================
```

Comment:
Will need to add: https://github.com/PlasmaPy/PlasmaPy/pull/742

Reply:
I started a project related to implementing PLEP 7, [here](https://github.com/PlasmaPy/PlasmaPy/projects/14), and placed the link in this PLEP.  As the associated PRs open, then I'll add those links too.